docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -529,4 +529,41 @@ Same as for [parseDateTimeBestEffort](#parsedatetimebesteffort) except that it r
 
 Same as for [parseDateTimeBestEffort](#parsedatetimebesteffort) except that it returns zero date or zero date time when it encounters a date format that cannot be processed.
 
+## toLowCardinality {#tolowcardinality}
+
+Converts input parameter to the [LowCardianlity](../data-types/lowcardinality.md) version of same data type.
+
+**Syntax**
+
+```sql
+toLowCardinality(expr)
```

Comment:
And if you want to convert from LC to ordinary type, you can use `CAST`. e.g. `CAST(x as String)`